unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/InternalECSComponentsId.cs

```diff
@@ -0,0 +1,5 @@
+public enum InternalECSComponentsId
+{
+    TEXTURIZABLE = 2,
```

Comment:
Any reason that is starting with 2 instead of 1?

Reply:
we currently have Transform as id 1. maybe we could change transform's id, we should talk about this with lean

Reply:
Maybe just by adding the `TRANSFORM = 1` in this same enum would be enough to clarify

Reply:
I could add a comment there if it helps clarifying, but in any case, there is a test here: https://github.com/decentraland/unity-renderer/blob/e48b52edf9fc89059fa6039a0cb2a224613cdce9/unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Tests/InternalComponentsIdShould.cs
that will prevent ids for colliding